.github/workflows/tagging.yml

```diff
@@ -0,0 +1,43 @@
+name: Tagging release
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - master
+jobs:
+  tagging-release:
+    name: Tagging release
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged && github.head_ref == 'prerelease' && github.base_ref == 'master'
+    steps:
+      - uses: actions/checkout@v2
+      - name: get commit message
+        shell: bash
+        run: |
+          echo ::set-env name=commitmsg::$(git log --format=%B -n 1 ${{ github.event.after }})
+          echo $commitmsg
+      - name: get tag version
+        run: |
+          echo ::set-env name=tagversion::$( sed -e 's#.*: \(\)#\1#' <<< ${commitmsg} )
+          echo $tagversion
+      - name: set tag version
+        id: set_tag_version
+        run: |
+          git config --local user.email "one.amex@aexp.com"
+          git config --local user.name "OneAmexBot"
+          echo ${tagversion}
+          git tag ${tagversion}
+          git push origin --tag
```

Comment:
```suggestion
          git push origin --tags
```

Reply:
Curious about this, is it an allowed operation to push a tag from the workflow?

Reply:
Seems like it is because the outcome of this workflow is the new tag that at the same time kicks the docker_release workflow

Reply:
Thanks for catching this, yes it is  @infoxicator yes that correct

Reply:
Thanks for catching this, i've updated it https://github.com/nellyk/one-app/runs/570463783?check_suite_focus=true and it seems to be working fine

weirdly, this seems to work `git push origin --tag` https://github.com/nellyk/one-app/runs/558700392?check_suite_focus=true

Reply:
Huh that is weird.. maybe it's pushing all local refs (which includes tags?) not present at the remote.